src/components/analytics/metrics-table.vue

```diff
@@ -56,6 +56,10 @@ export default {
   .metric-value {
     text-align: right;
   }
+
+  tbody {
+    border-bottom: 1px solid #ddd;
```

Comment:
We don't usually put a border below a table, so I feel unsure about this change. But if you feel like it's an improvement, I'd say feel free to run with it. But in that case, I think it'd be better to use the Sass variable `$border-top-table-data` in order to ensure that this border is the same as the one between rows.

Reply:
It looked like an odd amount of whitespace at first, but now when I try it without the line, I think it looks good! I guess I'll leave it out.